.travis.yml

```diff
@@ -46,23 +42,13 @@ before_install: |
     source multibuild/common_utils.sh
     source multibuild/travis_steps.sh
     before_install
-    wget https://mediaarea.net/download/binary/libmediainfo0/18.05/MediaInfo_DLL_18.05_Mac_i386+x86_64.tar.bz2
-    tar xf MediaInfo_DLL_18.05_Mac_i386+x86_64.tar.bz2
-  else
-    wget https://mediaarea.net/download/binary/libzen0/0.4.37/libzen0_0.4.37-1_amd64.xUbuntu_14.04.deb
-    wget https://mediaarea.net/download/binary/libmediainfo0/18.05/libmediainfo0_18.05-1_amd64.xUbuntu_14.04.deb
-    ar x libzen0_0.4.37-1_amd64.xUbuntu_14.04.deb
-    tar xf data.tar.xz
-    ar x libmediainfo0_18.05-1_amd64.xUbuntu_14.04.deb
-    tar xf data.tar.xz
+    curl https://mediaarea.net/download/binary/libmediainfo0/18.08.1/MediaInfo_DLL_18.08.1_Mac_i386+x86_64.tar.bz2 | tar xj
+    mv MediaInfoLib/libmediainfo.*dylib pymediainfo
```

Comment:
I'm not sure this is a good idea, isn't there a risk that it will be included in sdist if I'm not careful enough and I end up running the PyPI deploy hook in the MacOS environment?

Reply:
sdist ignores package_data where the dylib is put. And an additional license file included in the sdist should not be a problem.